### ISSUE TITLE
Luminosity ratio of the focus indicator for the table rows with background is less than required ratio 3:1: A11y_WinForm.NETRuntime_Non-TextContrast

### ISSUE THREAD

**Issue by kupatkar99** [dotnet/winforms#13995]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Keyboard; #A11yMAS;

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle. 
2. Splash screen appears and SSMA is launched
3. Navigate to the 'Tools' menu and select it.
4. Navigate to 'Project settings menu item and select it.
5. Navigate to "Type Mapping" button select it.
6. Navigate all controls under "Type Mapping" button.
7. Verify whether luminosity ratio for the focus indicator with background is less than required ratio or not.

### Actual Result:
Luminosity ratio of the focus indicator for the table rows with background is less than required ratio 3:1.

### Expected Result:
Luminosity ratio of the focus indicator for the table rows with background should satisfy the required ratio 3:1.

### User Impact:
Low vision users will be impacted. Luminosity contrast ratio for focus indicator is less than minimum required ratio i.e. 3:1. Due to this issue user will be unable to track the keyboard focus easily.

### Sample App used:-

[[[[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=M4uY2O)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=cCZLff)](https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-kkumar2_microsoft_com/ETnjjCF0jsZFiF_P9fDDmyIBlb80rXpmeIDu2or6768kVg?e=wBZc6R)

### Attachment:

<img width="882" height="481" alt="Image" src="https://github.com/user-attachments/assets/5f813a48-611d-4f06-9cb9-8bd4e7ddde77" />

**Comment by merriemcgaw** [comment]
It looks like the sample project is a .NET 9 project, @Olina-Zhang can you verify the behavior in .NET 10? @LeafShi1 we can take a fix for 10 but we wouldn't change earlier releases for this.

**Comment by LeafShi1** [comment]
The sample project is a .NET8.0 project

<img width="2294" height="1251" alt="Image" src="https://github.com/user-attachments/assets/4e791903-e592-4126-b1c5-faa88491a81a" />

This problem does not recur in the .net10.0 project

<img width="2096" height="1211" alt="Image" src="https://github.com/user-attachments/assets/08cc512f-bccb-4d88-a927-412a2acbbdf6" />

**Comment by Olina-Zhang** [comment]
@merriemcgaw  It cannot repro in .NET 10 & 9, was fixed by GH PR: https://github.com/dotnet/winforms/pull/10176.

**Comment by merriemcgaw** [comment]
@kupatkar99 - this is fixed in the current versions, and this issue can be closed.

**Comment by merriemcgaw** [comment]
@LeafShi1, they are migrating to .NET 8 and this is a grade blocking issue. Let's go ahead and service this for the next .NET 8 update. Please let me and @KlausLoeffelmann know if there was a substantial change between the two versions and we will evaluate risk.

**Comment by LeafShi1** [comment]
> [@LeafShi1](https://github.com/LeafShi1), they are migrating to .NET 8 and this is a grade blocking issue. Let's go ahead and service this for the next .NET 8 update. Please let me and [@KlausLoeffelmann](https://github.com/KlausLoeffelmann) know if there was a substantial change between the two versions and we will evaluate risk.

There are no major differences between the two versions. Please review the backport PR https://github.com/dotnet/winforms/pull/14001

**Comment by Zheng-Li01** [comment]
Verified the issue with .NET SDK 8.0.23 Test Pass build, the issue has been fixed as below screenshot.

<img width="1152" height="537" alt="Image" src="https://github.com/user-attachments/assets/f63c28b2-6546-4933-8327-c79b3702e50a" />

**Comment by Yash14j** [comment]
Issue is fixed in NET SDK 8.0.23 hence closing the bug.

#Closed;
